tools/sgen/Sources/SGen/Generatable/RhsValue.swift

```diff
@@ -124,6 +124,18 @@ enum RhsValue {
         }
     }
     
+    static func resolveRedirection(rhs: RhsValue) -> RhsValue? {
+        if rhs.isRedirect == false {
+            return rhs
+        }
+        
+        var redirection = rhs.redirection!
+        if !redirection.hasPrefix("mainProxy()") {
```

Comment:
where is the keyword `mainProxy` coming from?

Reply:
It's hard coded in sGen code.
It refers to the snippet below created for each appearance proxy:

`public let mainProxy: () -> FluentUIStyle`